### ISSUE TITLE
Vorfruchtwert, N-Saldo und Bilanz in Verbindung mit ungenutzten Zwischenfrüchten

### ISSUE THREAD

**Issue by simonkriegnerschramml** [w3geo/bml-fast#117]
Hallo Arnold,

Wie im Issue (#87) erwähnt eröffne ich hierzu ein eigenes Issue.

Vorfruchtwert:
Verbindung zu HF1: Vorfruchtwerte, welche durch den Inhalt aus der Zelle BAS15 getriggert werden, werden konkret auf die Hauptfrucht 1 gerechnet. Dieser Vorfruchtwert ist Teil des anrechenbaren Stickstoffs. Ebenso verringert er den Wert für die Düngeobergrenze.
Verbindung zu ungenutzten Zwischenfrüchten:
Vorfruchtwerte werden auch hier der Düngeobergrenze in Abzug gebracht. Ich darf die Zwischenfrucht eben um diesen Wert weniger Düngen. Damit ist der Vorfruchtwert auch Teil des anrechenbaren Stickstoffs der ungenutzten Zwischenfrucht. Jedoch darf dieser nicht in die Gesamtbilanz des Schlages (über alle Kulturen hinweg) einfließen, da dies ja bei der Hauptfrucht eingerechnet wird. Es würde ansonsten eine doppelte Anrechnung des Vorfruchtwertes geben.

Düngung:
Nährstoffmengen aus der Düngung zu ungenutzten Zwischenfrüchten müssen immer zu 100 % der folgenden Hauptkultur angerechnet werden. Am Besten man rechnet den Wert in den zur entsprechenden Düngekategorie der HF 1 hinzu.

N-Saldo:
Der N-Saldo aus BAS16 wird der ungenutzten Zwischenfrucht ebenso in Rechnung gestellt und von der Düngeobergrenze der ungenutzten Zwischenfrucht in Abzug gebracht. Jedoch werden von diesem, unter Berücksichtigung des Reduktionsfaktors (BAS10), nur mehr 60% bzw. 80% auf die folgende Hauptkultur weitergeschrieben. Dieser verringerte Wert, ist jener, der für die Gesamtbilanz schlagend wird. Der N-Saldo, der aus BAS16 stammt und auf die ungenutzte Zwischenfrucht angewendet wird, darf hier nicht einfließen.

LG
Simon

**Comment by arnold-pichler** [comment]
> N-Saldo

Frage:  Das bedeutet also, wenn es KEINE Zwischenfrucht gibt, wird der N-Saldo zu 100% schlagend, wenn es eine UNGENUTZTE gibt, nur mehr zu 60% bzw. 80% , je nach BAS10 ?

**Comment by arnold-pichler** [comment]
UND: Inwiefern betrifft das auch #109 ?

Denn da haben wir jetzt ja die Regel bei Gemüse-Früchten: Wenn NSaldo > NMin -> NSaldo, sonst NMin. Mit einer ungenutzten ZF müßte ich dann aber mit dem reduzierten NSaldo vergleichen, oder?

**Comment by simonkriegnerschramml** [comment]
> > N-Saldo
> 
> Frage: Das bedeutet also, wenn es KEINE Zwischenfrucht gibt, wird der N-Saldo zu 100% schlagend, wenn es eine UNGENUTZTE gibt, nur mehr zu 60% bzw. 80% , je nach BAS10 ?

Genau. eine ungenutzte ZWF ermöglicht die erneute Anwendung des Reduktionsfaktors (0,6 (Feuchtgebiet) oder 0,8 (Trockengebiet)). Auch wenn das fachlich nicht konsistent erklärt werden kann.
Da wir BAS16 so definiert haben, dass der anzurechnende N-Saldo, also das Produkt aus Bilanzüberschuss mal Reduktionsfaktor, angegeben wird, müssen wir die erste Multiplikation zur HF 1 nicht bedenken.

**Comment by simonkriegnerschramml** [comment]
> UND: Inwiefern betrifft das auch #109 ?
> 
> Denn da haben wir jetzt ja die Regel bei Gemüse-Früchten: Wenn NSaldo > NMin -> NSaldo, sonst NMin. Mit einer ungenutzten ZF müßte ich dann aber mit dem reduzierten NSaldo vergleichen, oder?

JA genau. Das ist aber in den Entscheidungsmatrizes so definiert. Also das beeinflusst die Entscheidung dahingehend, dass der N-Saldo der zur Gemüsekultur schlagend werden würde (also jener der auf die Gemüsekultur anzuwenden ist) mit dem Nmin abgeglichen werden muss, egal ob nun eine ZWF vorhanden ist oder nicht. Der N-Saldo, welcher mit dem Nmin verglichen werden muss, ist immer jener, der der Gemüsekultur zugerechnet werden müsste. Dieser ist halt höher, wenn es keine ZWF gibt und niedriger wenn es eine gibt.

Bsp.:
BAS16 (Feuchtgebiet)= 30
ungenutzte ZWF: NEIN
Nmin = 21
21<30: daher ist BAS16 der schlagende Wert

wenn aber

BAS16 (Feuchtgebiet)= 30
ungenutzte ZWF: JA
resultierender N-Saldo: 30 x 0,6 = 18
Nmin = 21
21>18: daher ist BAS16 der schlagende Wert


Bitte bedenke, dass bei HF1 auf HF2, dann aber der Bilanzüberschuss von >20 kg von Relevanz ist und nicht BAS16. Hier muss dann entschieden werden, ob überhaupt ein N-Saldo weiter berechnet werden muss, da ja erst ab mehr als 20 kg eine Berechnung stattfinden muss.


LG
Simon
